core/lib/spree/testing_support/factories/address_factory.rb

```diff
@@ -11,7 +11,7 @@
     end
 
     firstname 'John'
-    lastname 'Doe'
+    lastname nil
```

Comment:
I would prefer to leave this as is. Since the factories are designed to be included in other people's app's, some people may be relying upon `'Doe'` to still be the last name. I don't expect it will be a major issue, but there's no reason we can't leave this as it was.